apps/researcher/src/components/form/edtf/converter.test.ts

```diff
@@ -0,0 +1,83 @@
+import {expect} from '@jest/globals';
+import {fromDateString, toDateString} from './converter';
+
+describe('toDateString', () => {
+  it('returns an empty string when no date components are provided', () => {
```

Comment:
For consistency, also use the term 'parts' here (rather than 'components')? This also applies to the test underneath.